cypress.config.ts

```diff
@@ -11,9 +11,10 @@ export default defineConfig({
   viewportWidth: 1920,
   viewportHeight: 1080,
   e2e: {
-    baseUrl: 'https://kyberswap.com',
+    baseUrl: 'https://kyberswap-interface-1923.pr.kyberengineering.io/',
```

Comment:
How about: `window.location.href` ?

Reply:
It didn't work.

Reply:
How about use an enviroment variable which is passed by CI?

Reply:
ah, nvm, you already did that